clockwork.go

```diff
@@ -73,10 +111,15 @@ type fakeClock struct {
 	l sync.RWMutex
 }
 
-// sleeper represents a caller of After or Sleep
+// sleeper represents a waiting timer from NewTimer, Sleep, After, etc.
 type sleeper struct {
-	until time.Time
-	done  chan time.Time
+	until    time.Time
+	callback func(interface{}, time.Time)
+	arg      interface{}
+
+	ch   chan time.Time
```

Comment:
I would probably try to adhere to the same naming convention as in `time.Timer` and simply call this `C`